customTypes/otp/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'otp'
```

Comment:
Is this file still needed now you've switched to node-otp?

Reply:
Good point, @ronaldmansveld - I knew I'd forget to remove this :)

Reply:
Now removed